docs/machine-learning/tutorials/tensorflow-transfer-learning.md

```diff
@@ -0,0 +1,337 @@
+---
+title: Use ML.NET in a TensorFlow transfer learning scenario for image classification
```

Comment:
It's very unlikely our user base would understand what transfer learning scenario means and what value does that bring in. Leading from the scenario perspective here would help, rewording this title to "Creating a custom Image Classifier using TensorFlow and ML.NET" or "Re-train an existing TensorFlow Image Classifier with ML.NET to add new categories" would be a better title.